api/ping.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+package docker.swarmkit.v1;
+
+import "gogoproto/gogo.proto";
+import "plugin/plugin.proto";
+
+// Ping is the API provided by managers in the cluster to prove reachability over their
+// advertised address.
+service Ping {
```

Comment:
I dunno if we want to use https://github.com/grpc/grpc/blob/9cb96ba4f05a2e9df2ad038f24f53a8e5eafd9c3/doc/health-checking.md